### DISCUSSION TITLE
Timecode Display

### DISCUSSION THREAD

**Discussion by RikshaDriver** [gopro/labs#1122] [Feature Requests]
Using Labs we can set some QR codes to show current time up to seconds, but not frames/ms etc.

The LTC feature shows timecode briefly when LTC source is  plugged in but there is no way to show this permanently.

Request:
4CC code to overlay timecode (but not prominently) on the screen

Something like TCDS=1 which will then display in a small area on the top.

**Comment by dnewman-gpsw** [comment]
There isn't much screen space, and the top area is already used for $PMSG="" notifications. It will likely need to be in the center, but fainter maybe.
